.drone.yml

```diff
@@ -20,6 +20,21 @@ pipeline:
       matrix:
         NEED_CORE: true
 
+  install-from-url:
+    image: owncloudci/core
+    download_url: ${DOWNLOAD_URL}
+    core_path: /var/www/owncloud/server
+    pull: true
+    exclude: "apps/encryption"
```

Comment:
Note: the encryption app is bundled in the tarball already. So we could also do test runs using the bundled encryption code, rather than replacing with the "current master" encryption code.